_topic_maps/_topic_map.yml

```diff
@@ -1343,7 +1343,7 @@ Topics:
   File: ingress-controller-dnsmgt
   Distros: openshift-enterprise,openshift-origin
 - Name: Configuring the Ingress Controller endpoint publishing strategy
-  File: nw-ingress-controller-endpoint-publishing-strategies
+  File: nw-configuring-ingress-controller-endpoint-publishing-strategy
```

Comment:
Assembly not impacted by redirect as it's not [listed here](https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx#L27-L87).